software/firmware/ui.py

```diff
@@ -1,7 +1,7 @@
 """This module provides reusable UI components.
 """
 
-from europi import CHAR_HEIGHT, CHAR_WIDTH, b1, k1, oled
+from europi import CHAR_HEIGHT, CHAR_WIDTH, OLED_HEIGHT, b1, k1, oled
```

Comment:
[required] It appears that this isn't used and should be removed.

Reply:
The solution to the comment below uses `OLED_HEIGHT` so it's no longer redundant